libcrux-ml-kem/src/vector/portable/vector_type.rs

```diff
@@ -1,6 +1,6 @@
 use crate::vector::traits::FIELD_ELEMENTS_IN_VECTOR;
 
-/// Values having this type hold a representative 'x' of the ML-DSA field.
+/// Values having this type hold a representative 'x' of the Kyber field.
```

Comment:
This is surprising to me since this is not in the `kyber` module, just wanted to check that this is intended.

Reply:
Thanks, fixed.